include/libcrypto/aes.h

```diff
@@ -1,6 +1,37 @@
 #ifndef AES_H
 #define AES_H
 
+#include "ultratypes.h"
+
+typedef struct {
+    /* 0x00 */ unsigned char direction;
+    /* 0x04 */ int Nr;
+    /* 0x08 */ u32 rk[60];
+    /* 0xF8 */ u32 ek[60];
+} AesKeyInstance; // size = 0x1E8
+
+typedef struct {
+    /* 0x0 */ unsigned int mode;
+    /* 0x4 */ unsigned char IV[16];
+} AesCipherInstance; // size = 0x14
+
+void* memcpy(void* dst, void* src, u32 num);
```

Comment:
There's no `libc/string.h`?

Reply:
They implemented their own libc functions, it seems. We need to build with -fno-builtin